packages/iris-grid/src/AdvancedFilterCreatorSelectValue.tsx

```diff
@@ -1,17 +1,47 @@
 /* eslint react/no-did-update-set-state: "off" */
 import React, { PureComponent } from 'react';
-import PropTypes from 'prop-types';
 import { CSSTransition } from 'react-transition-group';
 import classNames from 'classnames';
+import { FilterCondition, Table } from '@deephaven/jsapi-shim';
 import Formatter from './Formatter';
 import TableUtils from './TableUtils';
 import AdvancedFilterCreatorSelectValueList from './AdvancedFilterCreatorSelectValueList';
 import './AdvancedFilterCreatorSelectValue.scss';
 
-class AdvancedFilterCreatorSelectValue extends PureComponent {
+interface AdvancedFilterCreatorSelectValueProps<T> {
+  invertSelection: boolean;
+  selectedValues: T[];
+  table: Table;
+  formatter: Formatter;
+  onChange: (selectedValues: T[], invertSelection: boolean) => void;
+  showSearch: boolean;
+  timeZone: string;
+}
+
+interface AdvancedFilterCreatorSelectValueState<T> {
+  error: unknown;
```

Comment:
In this class, we know the `error` is going to be a `string`. Make it optional here:
```suggestion
  error?: string;
```